Lib/fractions.py

```diff
@@ -451,8 +453,8 @@ def reverse(b, a):
 
     def _add(a, b):
         """a + b"""
-        na, da = a.numerator, a.denominator
-        nb, db = b.numerator, b.denominator
+        na, da = a._numerator, a._denominator
```

Comment:
Also in `def __eq__(a, b)` (around line 686) you can change `b.numerator` into `b._numerator`

Reply:
Thanks for the suggestion, but I doubt this will fork:
```
        if isinstance(b, numbers.Rational):
            return (a._numerator == b.numerator and
                    a._denominator == b.denominator)
```
As you can see, b is a Rational instance and it may lacks private `_numerator/_denominator` properties.